### ISSUE TITLE
Manjaro Linux not supported/cannot install

### ISSUE THREAD

**Issue by bissei** [saltstack/salt-bootstrap#778]
Hi,

Cannot install Salt to Manjaro. Please, add support for this. Thanks!

```
NAME="Manjaro Linux"
ID=manjaro
PRETTY_NAME="Manjaro Linux"
ANSI_COLOR="1;32"
HOME_URL="http://www.manjaro.org/"
SUPPORT_URL="http://www.manjaro.org/"
BUG_REPORT_URL="http://bugs.manjaro.org/"
/etc/manjaro-release
Manjaro Linux
/etc/lsb-release
DISTRIB_ID=ManjaroLinux
DISTRIB_RELEASE=15.12
DISTRIB_CODENAME=Capella
DISTRIB_DESCRIPTION="Manjaro Linux"

LSB Version:    n/a
Distributor ID: ManjaroLinux
Description:    Manjaro Linux
Release:    15.12
Codename:   Capella

 *  INFO: /bin/sh ./install_salt.sh -- Version 2015.11.09

 *  INFO: System Information:
 *  INFO:   CPU:          GenuineIntel
 *  INFO:   CPU Arch:     x86_64
 *  INFO:   OS Name:      Linux
 *  INFO:   OS Version:   4.4.5-1-MANJARO
 *  INFO:   Distribution: ManjaroLinux 15.12

 *  INFO: Installing minion
 *  INFO: Found function daemons_running
 * ERROR: No dependencies installation function found. Exiting...
```

**Comment by jfindlay** [comment]
@bissei, thanks for reporting.

**Comment by briancappello** [comment]
@bissei There's an easy work-around to get this working. You just gotta fool the script into thinking you're running Arch (because, in actuality, you are):

``` bash
sudo sed -i 's/ManjaroLinux/Arch/' /etc/lsb-release
sudo sh install_salt.sh [...]
sudo sed -i 's/Arch/ManjaroLinux/' /etc/lsb-release
```

**Comment by rallytime** [comment]
This should be fixed with #1063. As such, I will close this. Thanks!
